include/zephyr/drivers/led/lp50xx.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2020 Seagate Technology LLC
+ * Copyright (c) 2022 Grinn
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_LED_LP50XX_H_
+#define ZEPHYR_INCLUDE_DRIVERS_LED_LP50XX_H_
+
+#define LP50XX_COLORS_PER_LED	3
+
+#define LP5009_MAX_LEDS		3
+#define LP5012_MAX_LEDS		4
+#define LP5018_MAX_LEDS		6
+#define LP5024_MAX_LEDS		8
+#define LP5030_MAX_LEDS		10
+#define LP5036_MAX_LEDS		12
+
```

Comment:
Please add a comment to explain that LP5012 macros are compatible with LP5009, LP5024 macros with LP5018 and LP3036 macros with LP3030.

Reply:
OK